app/controllers/admin/authorised_email_domains_controller.rb

```diff
@@ -47,4 +49,16 @@ def destroy
   def authorised_email_params
     params.require(:authorised_email_domain).permit(:name)
   end
+
+  def sortable_columns
+    %w[name]
+  end
+
+  def sort_column
+    sortable_columns.include?(params[:sort]) ? params[:sort] : "name"
```

Comment:
While it's not a lot of duplication, would be good to use `sortable_columns` for the default:

```suggestion
    sortable_columns.include?(params[:sort]) ? params[:sort] : sortable_columns.first
```